builtin/game/item_entity.lua

```diff
@@ -75,7 +76,8 @@ core.register_entity(":__builtin:item", {
 			itemstring = self.itemstring,
 			always_collect = self.always_collect,
 			age = self.age,
-			dropped_by = self.dropped_by
+			dropped_by = self.dropped_by,
+			mergeable = self.mergeable
```

Comment:
You mean that `mergeable` should be `nil` by default, and if it is set to `true` it will not be saved in the static data?